app/controllers/charge_controller.js

```diff
@@ -50,9 +51,15 @@ const appendChargeForNewView = async function appendChargeForNewView(charge, req
   charge.allowApplePay = charge.gatewayAccount.allowApplePay
   charge.allowGooglePay = charge.gatewayAccount.allowGooglePay
   charge.googlePayGatewayMerchantID = charge.gatewayAccount.gatewayMerchantId
+  
+  const merchantId = getMerchantId(charge.gatewayAccount.gatewayAccountId)
```

Comment:
Can this maybe be `googlePayMerchantId`? On its own, `merchantId` is a bit non-specific.